types/common.tsx

```diff
@@ -1,7 +1,19 @@
 // Type aliases
 export type I18nString = string
 
-// Type interfaces for API gateway result
+// Types for TransLine component
+export interface TransLineProps {
+  i18nKey: I18nString
+  links?: string[] | null
```

Comment:
I think it was because I thought the `links` argument might get passed `null`. I'm currently initializing it in `TransLine.tsx` as null, but I think I can initialize it as `[]` instead.